depends/packages/native_ccache.mk

```diff
@@ -1,15 +1,23 @@
 package=native_ccache
-$(package)_version=3.7.12
+$(package)_version=4.6.3
```

Comment:
Is there any reason not to use 4.7.4 which is the latest release? https://github.com/ccache/ccache/releases

Reply:
I tried that, and it [broke Ubuntu 18.04 support](https://github.com/zcash/zcash/pull/5301#issuecomment-1379600293). I guessed that it was a problem with 4.7 and reverted to 4.6.3 which worked. We can either figure out the 4.7 issue in a subsequent PR, or just wait until Ubuntu 18.04 reaches EoL in April.